src/main/java/io/datanerds/avropatch/Patch.java

```diff
@@ -35,16 +45,29 @@ public boolean addOperation(Operation operation) {
         return Collections.unmodifiableList(operations);
     }
 
+    public Map<String, ?> getHeaders() {
```

Comment:
Not a concurrency issue with the map as long as the map is not open to modification. It might not be the case right now but leaves the code open to errors in case there is a write interface exposed to the map erroneously in the future. Would suggest initializing the map as immutable during initialization in the constructor.